src/workbook/interfaces.ts

```diff
@@ -17,6 +17,7 @@ export interface StockPlanModel {
   id?: string;
   plan_name: string;
   initial_shares_reserved?: string;
+  stock_class_id?: string;
```

Comment:
Important to find the stock class for a plan and check if that class is preferred or not